RepositoryCompiler/CodeModel/CodeParsers/CSharp/CaDETMemberMetricCalculator.cs

```diff
@@ -121,5 +137,404 @@ private int CountOccurrences(string text, string pattern)
             }
             return count;
         }
+
+        private int CountTryCatchBlocks(MemberDeclarationSyntax method)
```

Comment:
Based on this code, the metric will add +1 for a try block with a single catch and +1 for a try block with 10 catch statements. Is this intended?

Reply:
That is how it was implemented in the https://github.com/mauricioaniche/ck, so I followed that example.